tooltester/src/test/java/io/dockstore/tooltester/helper/DockstoreEntryHelperTest.java

```diff
@@ -43,6 +45,7 @@ public void generateLaunchToolCommand() {
         Assert.assertEquals(command, "dockstore tool launch --entry quay.io/briandoconnor/dockstore-tool-md5sum:1.0.4 --json test.json --script");
     }
 
+    @Ignore
```

Comment:
Might be helpful to add a message to why these tests are ignored since the test before says it shouldn't be ignored

Reply:
Sigh, left a debugging message in the PR. Thanks for catching it, I have removed that line.

I will add a line to the files though saying why so many tests are being ignored.

Reply:
Comment Added